syntax/syntax.go

```diff
@@ -6,6 +6,7 @@
 package syntax // import "go.starlark.net/syntax"
 
 // A Node is a node in a Starlark syntax tree.
+
```

Comment:
Nit: why the blank line? This should probably still be a doc comment.

Reply:
Why? Because of my inept fumbling with the keyboard. Fixed!